stream.js

```diff
@@ -73,8 +73,14 @@ function WebSocketStream(target, protocols, options) {
   if (socket.readyState === socket.OPEN) {
     stream = proxy
   } else {
-    stream = duplexify.obj()
-    socket.onopen = onopen
+    if (isBrowser) {
```

Comment:
you can collapse this into the previous else.

Reply:
You mean something like this:

  if (isNative && isBrowser) {
      socket = new WS(target, protocols)
      stream = proxy
      stream.cork()
      socket.onopen = onopenBrowser
    } else {
      socket = new WS(target, protocols, options)
      stream = duplexify.obj()
      socket.onopen = onopen
    }

Reply:
```js
if (socket.readyState === socket.OPEN) {
    stream = proxy
  } else if (isBrowser) {

} else {

}
```